packages/cli/src/cli.ts

```diff
@@ -47,10 +47,12 @@ const authorInformation: Promise<
 
 const getConfig = async (configPath = 'tsoa.json'): Promise<Config> => {
   let config: Config;
+  const ext = path.extname(configPath);
   try {
-    const ext = path.extname(configPath);
```

Comment:
if you would leave the assignment `ext = path.extname(configPath);` inside try-catch then it can be catched in line `61`: https://github.com/lukeautry/tsoa/blob/dcc225fd265fdacaab686d4ac359dde640663590/packages/cli/src/cli.ts#L61-L62

Reply:
`path.extname` doesn't actually try to find the file, and the only error it can throw is if you pass a non-string argument to it. (See https://nodejs.org/api/path.html#pathextnamepath).